add precision parameter to round function (#13965)
crate/crate: docs/appendices/release-notes/unreleased.rst

  :ref:`standard_conforming_strings <conf-session-standard_conforming_strings>`
  read-only session setting for improved compatibility with PostgreSQL clients.

> - Enhanced the :ref:`round <scalar-round>` scalar function to allow a second
>   input parameter for rounding precision.
> 

Reviewer: ```suggestion
- Added an optional ``precision`` parameter to the :ref:`round <scalar-round>` scalar function.

```
